src/app/people/people.component.ts

```diff
@@ -19,7 +19,9 @@ export class PeopleComponent implements OnInit {
   constructor(private HttpService: PeopleService) { }
 
   ngOnInit() {
-    this.people = this.HttpService.getPeople().map(data => data.results);
-    this.people.subscribe(() => this.showSpinner = false);
+    this.people = this.HttpService.getPeople().map(data => {
+      this.showSpinner = false;
```

Comment:
It's not a good idea to handle loading state in `map` function, avoid adding side effect